source/Tests/Playnite.Tests/Converters/PlayTimeToStringConverterTests.cs

```diff
@@ -0,0 +1,50 @@
+﻿using NUnit.Framework;
+using Playnite.Converters;
+using Playnite.SDK;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace Playnite.Tests.Converters
+{
+    [TestFixture]
+    public class PlayTimeToStringConverterTests
+    {
+        private static string Convert(object value, object formatToDaysParameter = null)
+        {
+            return PlayTimeToStringConverter.Instance.Convert(value, typeof(string), formatToDaysParameter, CultureInfo.CurrentCulture) as string;
+        }
+
+        [Test]
+        public void ConvertLocalizationStringsTest()
```

Comment:
This tests doesn't actually test anything because localization resources are not working in test projects. You are always comparing placeholder string with placeholder string. I have fixed that in devel so localization resource will work now.

Even if those string have worked however, you would be comparing the original string mask so the test would always fail.

The test should test against final strings, like this:
```
Assert.AreEqual("25h 20m", Convert(time, false))
```

Reply:
The test actually succeeded. I found after initially creating the test that localization resources didn't work, it just returned the default `<!LOCStringName>` string that gets obtained when the resource is not available. For this reason, both the Localization string in the test and what was obtained in the converter was that same not-found string that made them match.

Test reimplemented in https://github.com/JosefNemec/Playnite/pull/3317/commits/77e2a186d72af1c6c22bdfceb56337bb215e23bf